pkg/services/alerting/notifiers/victorops.go

```diff
@@ -0,0 +1,87 @@
+package notifiers
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/grafana/grafana/pkg/bus"
+	"github.com/grafana/grafana/pkg/log"
+	"github.com/grafana/grafana/pkg/metrics"
+	m "github.com/grafana/grafana/pkg/models"
```

Comment:
nit: I know import aliases can be helpful, but in this case I think aliasing "models" to "m" is a little excessive.